step.go

```diff
@@ -10,11 +10,14 @@ import (
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/env"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-steplib/bitrise-step-artifact-pull/api"
 	"github.com/bitrise-steplib/bitrise-step-artifact-pull/downloader"
 	"github.com/bitrise-steplib/bitrise-step-artifact-pull/model"
 )
 
+const relativeDownloadPath = "_tmp"
```

Comment:
According to `NormalizedOSTempDirPath`:
```
// NormalizedOSTempDirPath ...
// Creates a temp dir, and returns its path.
// If tmpDirNamePrefix is provided it'll be used
//  as the tmp dir's name prefix.
// Normalized: it's guaranteed that the path won't end with '/'.
```

So I think this should be `_artifact_pull` or something more specific (no need to add `tmp` as it's created by `ioutil.TempDir` which adds a random number to the dirname anyhow.

Also you can consider raneming the variable to `downloadDirPrefix` (as this is just a prefix and not the actual path)